.changeset/rotten-rules-switch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/common": patch
+---
+
+Added an optional `indexerUrl` property to `MUDChain` which is used as the default indexer URL in `createStoreSync`.
+
+Also added indexer URL's to the Redstone and Garnet chain configs.
```

Comment:
can we add a changelog for store-sync that talks about `createStoreSync` defaulting to `chain.indexerUrl` (or `false` to intentionally unset/not default)

and then this one can just focus on adding `indexerUrl` to `MUDChain` and filling it in for redstone, garnet chains